test/cypress/integration/widgets/filters_spec.ts

```diff
@@ -243,4 +269,433 @@ describe('Filters widget', () => {
+                cloneModalErrors.forEach(verifyErrorHandling);
+            });
+        });
+    });
+
+    describe('should allow to define all kinds of filter rules', () => {
+        const testPrefix = 'filters_test_form';
+        const blueprintId = `${testPrefix}_blueprint`;
+        const deploymentId = `${testPrefix}_deployment`;
+
+        before(() => {
+            cy.deleteDeployments(testPrefix)
+                .deleteBlueprints(testPrefix)
+                .uploadBlueprint('blueprints/empty.zip', blueprintId)
+                .deployBlueprint(blueprintId, deploymentId)
+                .setLabels(deploymentId, [
+                    { os: 'linux' },
+                    { os: 'windows' },
+                    { infra: 'aws' },
+                    { infra: 'gcp' },
+                    { infra: 'azure' }
+                ])
+                .deleteSites(testPrefix)
+                .createSites([
+                    { name: `${testPrefix}_Cracow` },
+                    { name: `${testPrefix}_Warsaw` },
+                    { name: `${testPrefix}_TelAviv` }
+                ]);
+        });
+
+        function selectRuleRowType(ruleRowType: FilterRuleRowType) {
+            withinLastRuleRow(() => {
+                cy.get('div[name="ruleRowType"]').click();
+                cy.get(`div[option-value="${ruleRowType}"]`).click();
+            });
+        }
+
+        function selectRuleOperator(operator: FilterRuleOperator) {
+            withinLastRuleRow(() => {
+                cy.get('div[name="ruleOperator"]').click();
+                cy.get(`div[option-value="${operator}"]`).click();
+            });
+        }
+
+        function selectRuleAttributeValues(
+            ruleRowType: FilterRuleRowType,
+            values: string[],
+            newValues = [] as string[],
+            withAutocomplete = false
+        ) {
+            if (values.length > 0) {
+                withinLastRuleRow(() => {
+                    const searchEndpoint: Record<FilterRuleRowType, string> = {
+                        blueprint_id: 'blueprints',
+                        site_name: 'sites',
+                        created_by: 'users',
+                        label: '' // NOTE: Only endpoints for attribute rules are necessary
+                    };
+                    cy.get('div[name="ruleValue"]')
+                        .click()
+                        .within(() => {
+                            values.forEach(value => {
+                                if (withAutocomplete) {
+                                    const searchEndpointRegExp = RegExp(
+                                        `${searchEndpoint[ruleRowType]}.*_search=${value}`
+                                    );
+                                    cy.interceptSp('GET', searchEndpointRegExp).as(`valueSearch_${value}`);
+                                }
+                                cy.get('input').type(`${value}`);
```

Comment:
If I'm not mistaken, we could use

```suggestion
                                cy.get('input').type(value);
```

The same applies in some other `type` calls :slightly_smiling_face:

Reply:
Sure, will change it.